test/evergreen.yml

```diff
@@ -3781,31 +3781,29 @@ buildvariants:
   run_on:
   - ubuntu2004-test
   expansions:
-    configure_env_vars:
-      CC=/opt/mongodbtoolchain/v3/bin/clang
-      CXX=/opt/mongodbtoolchain/v3/bin/clang++
-      PATH=/opt/mongodbtoolchain/v3/bin:$PATH
-      CFLAGS="-fsanitize=address -fno-omit-frame-pointer -ggdb"
-      CXXFLAGS="-fsanitize=address -fno-omit-frame-pointer -ggdb"
     posix_configure_flags:
-      --enable-silent-rules
-      --enable-strict
-      --enable-diagnostic
-      --disable-static
-      --enable-tcmalloc
-      --prefix=$(pwd)/LOCAL_INSTALL
+      -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_v3_clang.cmake
+      -DCMAKE_BUILD_TYPE=ASan
+      -DENABLE_STRICT=1
+      -DHAVE_DIAGNOSTIC=1
+      -DENABLE_STATIC=0
+      -DENABLE_TCMALLOC=1
+      -DCMAKE_PREFIX_PATH="$(pwd)/../TCMALLOC_LIB"
+      -DCMAKE_INSTALL_PREFIX=$(pwd)/LOCAL_INSTALL
+      -DCMAKE_C_FLAGS="-ggdb"
     smp_command: -j $(grep -c ^processor /proc/cpuinfo)
```

Comment:
Do we still need this?

Reply:
Yep, we need the `smp_command`, some of the sub-tasks in this variant (and others) make use of the `smp_command` variable.